tests/README.md

```diff
@@ -0,0 +1,32 @@
+# Tests for Curation
+
+These tests can be run with:
+
+```Shell
+$ ./tests/run_tests.sh
+```
+or with:
+```Shell
+$ PYTHONPATH=.:./data_steward:$PYTHONPATH python tests/runner.py --test-path <optional path>  --test_pattern <optional pattern> --coverage-file <required coverage file path> 
+```
+
+## Directory Structure
```

Comment:
A few lingering files/directories I see in this PR which look to be misplaced:
- https://github.com/all-of-us/curation/tree/lb/testing-changes/data_steward/test/unit_test/data_steward/cdr_cleaner
- https://github.com/all-of-us/curation/tree/lb/testing-changes/tests/unit_tests/data_steward/data_steward/curation_logging

Reply:
The files at the old location have been added and merged since creating this PR, so the developers put them at the "old location relative to this PR".  I rebased because of merge conflicts and got these.  I'll have to look and see if they are really unit tests or integration tests.  

There's been something weird going on with the `curation_logging` package.  It also messes up when I rebase.  I'll fix it again and it shouldn't be a problem once merged.

Reply:
Had to merge some more test changes in just now.  Rebuilding.